.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   - "0.10"
 before_script:
   - cp config.js.example config.js
+  - npm run nsp
```

Comment:
This is run twice, as `npm run nsp` is also part of `npm test`.

On the other hand, the dependencies will **always** be up-to-date (against their major versions, which are never vulnerable) on the Travis CI instances, as it builds an environment from scratch using the latest packages.